package-lock.json

```diff
@@ -907,6 +907,16 @@
         "@babel/helper-plugin-utils": "^7.8.3"
```

Comment:
We can delete this no?

Reply:
Part of package lock, related to a bigger question of do we want package.lock or not.